src/styles/variables.scss

```diff
@@ -0,0 +1,20 @@
+$card-title-size: 12px;
+$card-title-weight: 500;
+$card-font-color: #060b35;
+$card-width: 200px;
+$img-width: 160px;
+$img-height: 134;
```

Comment:
It seems that the unit of measurement is missing for the `$img-height` variable. It's important to specify units for consistency and to avoid potential issues in rendering. You should add a unit like `px` to this value.